examples/basic-jsx-precompile/index.html

```diff
@@ -17,9 +17,26 @@
     </div>
     <h4>Example Details</h4>
     <p>This is written with JSX in a separate file and precompiled to vanilla JS by running:</p>
+
+    <p><i>With Babel lower than 6.0</i></p>
     <pre>npm install -g babel
 cd examples/basic-jsx-precompile/
 babel example.js --out-dir=build</pre>
+
+	<p><i>With Babel 6.0 or higher</i></p>
+	<p>In the example folder, create a file called</p>
+	<pre>.babelrc</pre>
+	<p>with the following contents:</p>
+	<pre>
+{
+  "presets": ["react"]
+}
+	</pre>
+	<p>Then run:</p>
+    <pre>npm install -g babel-cli
+cd examples/basic-jsx-precompile/
+npm install babel-preset-react
+babel example.js --out-dir=build</pre>
```

Comment:
We can skip the creation of the `.babelrc` file to cut out a step and just put `babel --presets react example.js --out-dir=build`